newsfragments/9109.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Stop spinners with check mark.
+
+Previously they stopped on whatever part of the animation.
```

Comment:
Please try to keep those news fragments single-line, as they end up in this bullet list:
https://docs.parsec.cloud/en/stable/history.html

Also mention that those spinners are from the CLI